.gitignore

```diff
@@ -52,10 +52,15 @@ Thumbs.db
 *.mov
 *.wmv
 
+# Vim specific configuration
 .vim
 
 # stockwatch data
 stocks/
 
 # Playground folder for local develop scripts
 develop/
+
+# The generated coverage report
+coverage.xml
```

Comment:
Line 33 is already meant for this I think; merge this new line with that line (on that location)